packages/file-upload/src/FileUploadRejectedFileList.tsx

```diff
@@ -0,0 +1,30 @@
+import { Stack } from '@ag.ds-next/box';
+import { FileUploadRejectedFile } from './FileUploadRejectedFile';
+
+export type RejectedFile = {
+	fileName: string;
+	fileSize: number;
+	errors: { message: string; code: string }[];
+};
+
+export const FileUploadRejectedFileList = ({
+	fileRejections,
+	handleRemoveRejection,
+}: {
+	fileRejections: RejectedFile[];
+	handleRemoveRejection: (fileName: string) => void;
+}) => {
+	if (fileRejections.length === 0) return null;
```

Comment:
I think this check should be moved back into the main `FileUpload` component. There's something about simple sub components like this that  can render `null` that doesn't sit well with me. But maybe lets chat about this.